src/layouts/app/translations-mixin.ts

```diff
@@ -60,12 +61,15 @@ export default (superClass: Constructor<LitElement & HassBaseEl>) =>
 
     private _loadTranslationFragment(panelUrl) {
```

Comment:
I feel like we should pass language in here too? It's weird that we're assuming the language to use on line 65, which again is a slight different variation on other usages.